pinax/stripe/tests/test_actions.py

```diff
@@ -984,6 +1005,36 @@ def setUp(self):
             stripe_id="cus_xxxxxxxxxxxxxxx"
         )
 
+    def test_sync_coupon_from_stripe_data(self):
+        account = Account.objects.create(
+            stripe_id="acct_X",
+            type="standard",
+        )
+        coupon = {
+            "id": "35OFF",
+            "object": "coupon",
+            "amount_off": None,
+            "created": 1391694467,
+            "currency": None,
+            "duration": "repeating",
+            "duration_in_months": 3,
+            "livemode": False,
+            "max_redemptions": None,
+            "metadata": {
+            },
+            "percent_off": 35,
+            "redeem_by": None,
+            "times_redeemed": 1,
+            "valid": True
+        }
+        coupons.sync_coupon_from_stripe_data(coupon)
+        c1 = Coupon.objects.get(stripe_id=coupon["id"], stripe_account=None)
+        self.assertEquals(c1.percent_off, decimal.Decimal(35.00))
```

Comment:
Could use/check the return value from `sync_coupon_from_stripe_data`.  Also below.